### PR TITLE
3CB Factions Wrong Magazines Fix

### LINKED ISSUE #3218: Validation errors of 3CB templates (Post Mag Well update?)
Several factions now attempt to load incompatible magazines into their guns after 3CB sorted out their mag well stuff

[Arma3_x64_2024-04-13_19-14-35.txt](https://github.com/official-antistasi-community/A3-Antistasi/files/14968190/Arma3_x64_2024-04-13_19-14-35.txt)

### PR DESCRIPTION
**Pull request by ante185**
## What type of PR is this.
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
Replaces incompatible magazines with compatible ones

### Please specify which Issue this PR Resolves.
closes #3218

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:

### CODE REVIEW

File: A3A/addons/core/Templates/Templates/3CB/3CB_AI_ION_Arctic.sqf [hunk 1]
```diff
@@ -219,9 +219,9 @@ _sfLoadoutData set ["rifles", [
 ["UK3CB_FAMAS_F1", "rhsusf_acc_rotex5_grey", "rhsusf_acc_anpeq15A", "rhsusf_acc_g33_xps3", ["UK3CB_FAMAS_25rnd_556x45"], [], ""]
 ]];
 _sfLoadoutData set ["carbines", [
-["UK3CB_G36A1_KSK_K", "rhsusf_acc_rotex5_grey", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_eotech_552", ["rhs_mag_30Rnd_556x45_Mk318_PMAG"], [], ""],
-["UK3CB_G36A1_KSK_K", "rhsusf_acc_rotex5_grey", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_eotech_xps3", ["rhs_mag_30Rnd_556x45_Mk318_PMAG"], [], ""],
-["UK3CB_G36A1_KSK_K", "rhsusf_acc_rotex5_grey", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_compm4", ["rhs_mag_30Rnd_556x45_Mk318_PMAG"], [], ""],
+["UK3CB_G36A1_KSK_K", "rhsusf_acc_rotex5_grey", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_eotech_552", ["rhssaf_30rnd_556x45_SOST_G36", "rhssaf_30rnd_556x45_EPR_G36"], [], ""],
```
Comment (ante185) [inline A3A/addons/core/Templates/Templates/3CB/3CB_AI_ION_Arctic.sqf:222]: 3CB factions already has SAF as a requirement

File: A3A/addons/core/Templates/Templates/3CB/3CB_AI_ION_Arctic.sqf [hunk 2]
```diff
@@ -231,10 +231,10 @@ _sfLoadoutData set ["grenadeLaunchers", [
...
-["UK3CB_AG36_KSK", "rhsusf_acc_nt4_black", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_su230_mrds", ["rhs_mag_30Rnd_556x45_Mk318_PMAG"], ["rhs_mag_M441_HE", "rhs_mag_M397_HET", "rhs_mag_M433_HEDP", "rhs_mag_m714_White"], ""]
+["UK3CB_AG36_KSK", "rhsusf_acc_nt4_black", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_eotech_552", ["rhssaf_30rnd_556x45_SOST_G36"], ["rhs_mag_M441_HE", "rhs_mag_M397_HET", "rhs_mag_M433_HEDP", "rhs_mag_m714_White"], ""],
+["UK3CB_AG36_KSK", "rhsusf_acc_nt4_black", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_ACOG_RMR", ["rhssaf_30rnd_556x45_SOST_G36"], ["rhs_mag_M441_HE", "rhs_mag_M397_HET", "rhs_mag_M433_HEDP", "rhs_mag_m714_White"], ""],
+["UK3CB_AG36_KSK", "rhsusf_acc_nt4_black", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_su230", ["rhssaf_100rnd_556x45_EPR_G36"], ["rhs_mag_M441_HE", "rhs_mag_M397_HET", "rhs_mag_M433_HEDP", "rhs_mag_m714_White"], ""],
+["UK3CB_AG36_KSK", "rhsusf_acc_nt4_black", "rhsusf_acc_anpeq15_bk", "rhsusf_acc_su230_mrds", ["rhssaf_100rnd_556x45_EPR_G36"], ["rhs_mag_M441_HE", "rhs_mag_M397_HET", "rhs_mag_M433_HEDP", "rhs_mag_m714_White"], ""]
```
Comment (jaj22) [inline A3A/addons/core/Templates/Templates/3CB/3CB_AI_ION_Arctic.sqf:237]: Are you intentionally using the 100rnd mags here?

Comment (ante185) [reply A3A/addons/core/Templates/Templates/3CB/3CB_AI_ION_Arctic.sqf]: no, fixed

**Review by jaj22 — APPROVED** [review]
Ok, seems fine.

**Pull request merged**